docs/developers/developers-setup.md

```diff
@@ -165,10 +165,10 @@ meson test
 
 #### Install on system and run
 
-To install Diffuse on your system (e.g. `/usr/local/`):
+To install Diffuse on your system (e.g., `/Applications` and `/opt/homebrew` or `/usr/local/`):
 
 ```sh
-meson install # requires admin privileges
+meson install # might require admin privileges
```

Comment:
Since you wrote this:

> [It] worked without sudo because I'm a member of the "admin" group on my system

I think you can revert to "requires", because no matter if it asks the question or not, it does require the user to have the admin privileges.